blocks/block-controls/index.js

```diff
@@ -8,10 +8,11 @@ import { Fill } from 'react-slot-fill';
  */
 import { Toolbar } from 'components';
 
-export default function BlockControls( { controls } ) {
+export default function BlockControls( { controls, children } ) {
 	return (
 		<Fill name="Formatting.Toolbar">
 			<Toolbar controls={ controls } />
```

Comment:
Technically it is optional, and `<Toolbar />` will not render itself when omitted:

https://github.com/WordPress/gutenberg/blob/97ee1a3/components/toolbar/index.js#L13-L15

Example: Included Text and Quote block changes render `<BlockControls />` without `controls` prop.